File: 2264-largest-3-same-digit-number-in-string/2264-largest-3-same-digit-number-in-string.py
class Solution:
    def largestGoodInteger(self, num: str) -> str:
        ans = " "
        for i in range(len(num) - 2):
            if num[i] == num[i+1] and num[i+1] == num[i+2]:
                if num[i] > ans[0]:
                    ans = num[i]*3
        return ans if ans != " " else ""

File: 2264-largest-3-same-digit-number-in-string/2264-largest-3-same-digit-number-in-string.java
class Solution {
    public String largestGoodInteger(String num) {
        char[] n = num.toCharArray();
        char[] ans = {' ', ' ', ' '};
        for(int i = 0; i < n.length - 2; i++){
            if(n[i] == n[i+1] && n[i+1] == n[i+2] && n[i] > ans[0]){
                ans[0] = n[i];
                ans[1] = n[i];
                ans[2] = n[i];
            }
        }
        return ans[0] == ' ' ? "" : new String(ans);
    }
}